iOS/rainbow/Controller/MainTabBarController.swift

```diff
@@ -73,7 +73,7 @@ class MainTabBarController: UITabBarController {
                         do {
                             try ScoreEntry.ClientPersistence.save(entry: newEntry)
                         } catch let saveError {
-                            SVProgressHUD.showError(withStatus: "Could not save user")
+                            SVProgressHUD.showError(withStatus: "You'll need an internet connection to get on the leaderboard!")
```

Comment:
this is saving in core data. so error message could be different

Reply:
Good catch.